### PR TITLE
chore(docs): Release notes 2021_16

### LINKED ISSUE #6688: Chore: Release notes 2021.16
**Describe the issue. What is the expected and unexpected behavior?**
Release notes 2021.16

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #6688

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,72 @@ id: Release notes
...
+  - Added support for classname ([#6647](https://github.com/patternfly/patternfly-react/pull/6647))
+- **Alert:**
+  -  Allowed all heading levels for title ([#6628](https://github.com/patternfly/patternfly-react/pull/6628))
+- **Alert:**
+  - Converted examples to functional with typescript ([#6634](https://github.com/patternfly/patternfly-react/pull/6634))
+- **Card:**
+  - Added disabled card and cleaned up props ([#6659](https://github.com/patternfly/patternfly-react/pull/6659))
+  -  Updated card demos to use select instead of dropdown ([#6639](https://github.com/patternfly/patternfly-react/pull/6639))
+- **Context selector:**
+  - Addedd support for focus trap to be made inactive ([#6638](https://github.com/patternfly/patternfly-react/pull/6638))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:31]: Removed extra "d"
```suggestion
  - Added support for focus trap to be made inactive ([#6638](https://github.com/patternfly/patternfly-react/pull/6638))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,72 @@ id: Release notes
...
+- **Date picker:**
+  - Updaetd to allow date picker users to open the calendar programatically ([#6632](https://github.com/patternfly/patternfly-react/pull/6632))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:33]: Spelling typos
```suggestion
  - Updated to allow date picker users to open the calendar programmatically ([#6632](https://github.com/patternfly/patternfly-react/pull/6632))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,72 @@ id: Release notes
...
+- **Pagination:**
+  - Updated `toggleTemplate` description to include `ofWord` prop ([#6622](https://github.com/patternfly/patternfly-react/pull/6622))
+- **Popover:**
+  - Added composable header and alert variants ([#6664](https://github.com/patternfly/patternfly-react/pull/6664))
+  - Forwarded focus to close button with voice over click ([#6660](https://github.com/patternfly/patternfly-react/pull/6660))
+- **Select:**
+  - Added ability to persist filtered children state ([#6670](https://github.com/patternfly/patternfly-react/pull/6670))
+  - Fixed no autocomplete for typeahead select ([#6623](https://github.com/patternfly/patternfly-react/pull/6623))
+- **Slider:**
+   Fixed examples with custom steps and input ([#6662](https://github.com/patternfly/patternfly-react/pull/6662))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:51]: Missing bullet point
```suggestion
  - Fixed examples with custom steps and input ([#6662](https://github.com/patternfly/patternfly-react/pull/6662))
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
A few nits

**Review by evwilkin — APPROVED** [review]
LGTM!

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**